code/game/gamemodes/objective_items.dm

```diff
@@ -114,6 +114,13 @@
 /datum/objective_item/steal/supermatter/TargetExists()
 	return GLOB.main_supermatter_engine != null
 
+//extra objectives for antagonists. [FULP]
+/datum/objective_item/steal/car_budget //cargonia's card good luck

```

Comment:
Not a fan of this change. If the cargo budget card gets compromised, the entire department becomes virtually useless. The traitor (or the Syndicate) can siphon all funds from the budget and now no one can buy anything unless they pay out-of-pocket, which is way too expensive even for most heads of staff.

Reply:
why would that be unacceptable?

Reply:
Because it encourages traitors to steal something that can fuck over an entire department.

As a cargonian, what are you supposed to do if someone steals the budget card and siphons money out of it every time you sell something? You have no way of tracking down where the card is or identifying who the thief is (if they've been smart and haven't left fingerprints everywhere), so you're just kinda screwed.

Reply:
Unlike other departmental high-risk items, like the HoS' laser gun or the CMO's hypospray, the cargo budget is essential to the entire department, and without it it cannot function. No other objective (besides hijack, I suppose) does that. Given other departments could still function even when their areas are destroyed (medical can still heal and do surgery outside of medical, security is usually outside of security anyway, engineering can always move somewhere else like auxilary power, etc.), giving traitors the objective to completely disable an entire department seems completely unbalanced.

Reply:
I’ll remove it whenever I get the chance to later tonight after work.